src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Defaulting and Validation of app CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [app CRs]({{< relref "/src/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}} is enabled. This logic is provided by
```

Comment:
This one should work

```suggestion
defaulting and validation logic of [app CRs]({{< relref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
```

Reply:
@ubergesundheit Thanks for checking. First time I've used relref and its very handy. 👍